bundles/org.openhab.binding.ws980wifi/src/main/java/org/openhab/binding/ws980wifi/internal/ws980wifiConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ws980wifi.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link ws980wifiConfiguration} class contains fields mapping thing configuration parameters.
+ * Configuration parameters have an individual value for each thing. In contrast property values are
+ * the same for all Things of one Thing-Type.
+ * The configuration parameters can be modified via the administration interfac. properties not.
+ * 
+ * @author Joerg Dokupil - Initial contribution
+ */
+@NonNullByDefault
+public class ws980wifiConfiguration {
+
+    public int refreshInterval;
```

Comment:
If you produced getter methods consider to make these fields private.

Reply:
done